e2e/cypress/integration/messaging/direct_messages_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import users from '../../fixtures/users.json';
+
+const userActive = users['user-1'];
+
+describe('Messaging', () => {
+    beforeEach(() => {
+        // # Create new user that will eventually be inactive
+        cy.apiLogin('sysadmin');
+        cy.apiCreateNewUser().then((user) => {
+            cy.wrap(user).as('userInactive');
+        });
```

Comment:
Short form would be,
```suggestion
        cy.apiCreateNewUser().as('userInactive');
```
Otherwise, you're repeating `.then` - one here and the other when [getting the value](https://github.com/mattermost/mattermost-webapp/pull/5591/files#diff-da2836c9fe5c05bd0e6d2eccc32efb08R29).

Reply:
I kept the `then` construct here to extract `user` from the response (now the `user` returned from `apiCreateNewUser` is wrapped)